client_code/MultiSelectDropDown/Option/__init__.py

```diff
@@ -82,6 +83,15 @@ def disabled(self, val):
         else:
             del self._dom_node.dataset["disabled"]
 
+    @property
+    def foreground(self):
+        return self._foreground
+
+    @foreground.setter
+    def foreground(self, value):
+        self._foreground = value
+        self.label.foreground = self.icon_checked.foreground = value
+
```

Comment:
Can we follow the code style of the component and keep props in the `_props` dict. 

Same for other components in this PR.

Reply:
done